src/modules/hacker/data/Ingverbs.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace faker {
+    const std::vector<std::string> ingverbs = {
+            "backing up",
+            "bypassing",
+            "hacking",
+            "overriding",
+            "compressing",
+            "copying",
+            "navigating",
+            "indexing",
+            "connecting",
+            "generating",
+            "quantifying",
+            "calculating",
+            "synthesizing",
+            "transmitting",
+            "programming",
+            "parsing",
+    };
+}
```

Comment:
missing new-lines in the end of line, also in other files, there are some settings in IDE like vscode, clion that could do that automatically for you